test/Infer/demanded-bits8.opt

```diff
@@ -0,0 +1,13 @@
+; REQUIRES: solver, solver-model
+
+; RUN: %souper-check %solver -infer-rhs -souper-infer-iN %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: result
+
+%0 = block 2
+%1:i32 = var
+%2:i32 = and 268435455:i32, %1
+%3:i64 = zext %2
+%4:i64 = phi %0, %3, %3
```

Comment:
do you mean to have both branches of the phi coming from the same place? can you make a better test case using phi where the phi is doing something non-trivial?

Reply:
Updated.